comp/metadata/host/hostimpl/utils/host.go

```diff
@@ -184,7 +185,7 @@ func GetPayload(ctx context.Context, conf config.Reader) *Payload {
 		PythonVersion: python.GetPythonInfo(),
 		SystemStats:   getSystemStats(),
 		Meta:          meta,
-		HostTags:      GetHostTags(ctx, false, conf),
+		HostTags:      hosttags.GetHostTags(ctx, false, conf),
```

Comment:
Possibly not worth the hassle, but `GetHostTags` could possibly be renamed `Get` now.